src/picongpu/include/particles/particleToGrid/ComputeGridValuePerFrame.def

```diff
@@ -143,7 +219,22 @@ struct CreateMacroCounterOperation
     typedef FieldTmpOperation< MacroParticleCounter, T_Species > type;
 };
 
-/* EnergyDensity */
+/** Energy Density Operation for Particle to Grid Projections
+ *
+ * Derives a scalar field for average kinetic particle energy per cell times the
+ * particle density from a particle species at runtime.
```

Comment:
nope, the term energy density is quite well defined to be kinetic. If someones does not get that in the title the brief directly explains it in detail.